recipes/resolvo-cpp/bld.bat

```diff
@@ -0,0 +1,9 @@
+@echo on
+
+cmake %CMAKE_ARGS% -GNinja %SRC_DIR%
```

Comment:
Shouldn't `Rust_CARGO_TARGET` be set as well?
```suggestion
cmake %CMAKE_ARGS% -GNinja %SRC_DIR% -DRust_CARGO_TARGET=%CARGO_BUILD_TARGET%
```

Reply:
With the previous remark about the build type, the suggestion above gives:
```suggestion
cmake ^
      -GNinja %SRC_DIR% ^
      -DRust_CARGO_TARGET=%CARGO_BUILD_TARGET% ^
      -DCMAKE_BUILD_TYPE=Release ^
      %CMAKE_ARGS% 
```

Reply:
`-DRust_CARGO_TARGET=%CARGO_BUILD_TARGET%` is only necesary when cross compiling which we arent currently doing on windows. But I will add it for future self.